src/api/v1/project/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+import { CreateProjectController, GetProjectsController, getSingleProject } from './project_controllers'
+import { isAuthenticated, isMentor } from '../../../middlewares/auth'
+
+const router = express.Router()
+
+router.post('/project', isMentor, CreateProjectController)
+router.get('/projects', isAuthenticated, GetProjectsController)
+router.get('/project/:projectId>', isAuthenticated, getSingleProject)
```

Comment:
This should be uniform 
```suggestion
router.get('/project', isAuthenticated, GetProjectsController)
router.get('/project/:projectId', isAuthenticated, getSingleProject)
```

Reply:
or all can be `/projects`